packages/compiler-sfc/__tests__/cssVars.spec.ts

```diff
@@ -103,8 +103,8 @@ describe('CSS vars injection', () => {
     const { content } = compileSFCScript(
       `<script>const a = 1</script>\n` +
         `<style>div{
-          color: v-bind(color);
-          font-size: v-bind('font.size');
+          color: v-bind(color, blue);
```

Comment:
I think it might be better to add new tests for this feature, rather than modifying the existing tests. Otherwise we lose tests for the case where a default isn't provided.

Reply:
I have already re-modified and rebased my own branch